mne/io/eeglab/eeglab.py

```diff
@@ -259,12 +289,14 @@ def read_raw_eeglab(input_fname, eog=(), preload=False,
     .. versionadded:: 0.11.0
     """
     return RawEEGLAB(input_fname=input_fname, preload=preload,
-                     eog=eog, verbose=verbose, uint16_codec=uint16_codec)
+                     eog=eog, uint16_codec=uint16_codec,
+                     montage_units=montage_units, verbose=verbose)
 
 
 @fill_doc
 def read_epochs_eeglab(input_fname, events=None, event_id=None,
-                       eog=(), verbose=None, uint16_codec=None):
+                       eog=(), uint16_codec=None, montage_units='mm',
```

Comment:
We have been adding these at a reasonable place. And this one's quite reasonable since the order has been (rightly) changed
```suggestion
                       eog=(), *, uint16_codec=None, montage_units='mm',
```